components/support/nimbus-fml/fixtures/fe/misc-features.yaml

```diff
@@ -11,12 +11,14 @@ channels:
   - debug
 features:
   onboarding:
-    description: A feature containing a field with pref-key
+    description: A feature containing a field with pref
     variables:
       enabled:
         description: If true, enable new style onboarding.
         type: Boolean
-        pref-key: enrollment_enabled
+        pref:
+          key: enrollment_enabled
+          branch: default
```

Comment:
pref-key is something else, unfortunately named, and we cant get rid of this. I dont think we should overload it.